source/dub/dependency.d

```diff
@@ -667,14 +666,15 @@ struct Version {
 	this(string vers)
 	{
 		enforce(vers.length > 1, "Version strings must not be empty.");
-		if (vers[0] != branchPrefix && vers != UNKNOWN_VERS)
+		if (vers[0] != branchPrefix && vers !is UNKNOWN_VERS)
```

Comment:
Is `UNKNOWN_VERS` internal-only ? If so you could make it `static immutable` and only compare `ptr`